app/queries/servers/channel.ts

```diff
@@ -99,7 +99,7 @@ export const prepareDeleteChannel = async (channel: ChannelModel): Promise<Model
     const relations: Array<Relation<Model>> = [channel.membership, channel.info, channel.settings];
     for await (const relation of relations) {
         try {
-            const model = await relation.fetch();
+            const model = await relation?.fetch?.();
```

Comment:
I haven't seen this before `?.()` - does the dot have any significance? I assume it's to call `fetch()` if the relation and the fetch method exist.

Reply:
That is used to call the function only if the function is not null or undefined